### ISSUE TITLE
Tables and lists accessibility work

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#30813]
## Issue Description

This epic is for the work to ensure VA.gov is using tables properly on our pages so that the information is accessible to all users. There are many instances when a list should be used instead of a table. And when we do use tables, we need to ensure we have proper headings, which are needed for screen readers.

Josh Kim's notes from a 9/30/21 meeting where we discussed the different use cases for tables vs. lists:

- We recommend providing guidance to editors to consider thinking from a mobile first layout before a desktop layout.
- Non-matrix tables with only one header type (either column or row) should be written as lists with [headings](https://www.w3.org/WAI/tutorials/page-structure/headings/).
For example, [the tables on survivor dlc rates should be lists](https://www.va.gov/disability/survivor-dic-rates/). See the first screenshot for an example of headings. [I used headingsmap to display the order.](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en)
- Matrix tables with both column and row headers should be written as tables.
For example, [the special monthly compensation rates tables](https://www.va.gov/disability/compensation-rates/special-monthly-compensation-rates/) should remain as tables as it's easier for screen reader users to parse through. They will first identify the row relevant to them, and then navigate right to determine the values.

**Additional notes:**
- [Zebra stripes help users visually group rows](https://adrianroselli.com/2017/11/a-responsive-accessible-table.html?replytocom=87367#GeneralStylesZebra) that change into blocks as users rotate screens or otherwise change context. This may be helpful particularly to folks who use screen magnification tools.
- [Headings on the special monthly compensation rates page](https://www.va.gov/disability/compensation-rates/special-monthly-compensation-rates/) should be given an accessibility review. Accordions may need to be listed as headings for them to be discoverable to screen reader users. They are also caption elements that _look_ like headings, but aren't.
